### ISSUE TITLE
eskk doesn't seem to find some words

### ISSUE THREAD

**Issue by x-yuri** [tyru/eskk.vim#232]
I can see in `SKK-JISYO.L`:

```
やさしi /優し;※送り位置補正/易し;※送り位置補正/
やさs /優;(gentle)優しい人/易;(easy)易しい問題/
```

But when I do `a<C-j>;yasashii<Space>` it doesn't find candidates. The same goes for `yasashi`, and `yasa`. I can see that the words in the dictionary end with `i`, and `s`. Can you possibly explain what that means?

I investigated it a bit. It appears there are okuri-ari and okuri-nasi entries there. Again not sure what that means. And okuri-ari entries are [in reverse order][1]. Okuri-ari entries take about 3% of the dictionary, so I'm not sure if they are reachable. But even if they would, those latin endings would most likely not let `eskk` find a match. On second thought if they would be reachable, I could probably use `<C-x><C-o>`. But they aren't. Is there a way to enter 易? 易しい is supposedly a common word, so I wonder what I'm doing wrong. Is `SKK-JISYO.L` not good enough? Or am I missing something?

[1]: https://gist.github.com/x-yuri/bed2178f16187791b9377928d5f0370d

**Comment by k-takata** [comment]
SKK is a unique Japanese input system. The user needs to specify the start point of Kanji and Hiragana (Okurigana) explicitly by using capital letters. See the [Japanese Wikipedia entry of SKK](https://ja.wikipedia.org/wiki/SKK) for detail.
To enter `易しい`, you need to type `YasaShii`, where the `Y` specifies the start point of Kanji and the `S` specifies the start point of Hiragana (Okurigana).

**Comment by x-yuri** [comment]
Actually, one might say that I'm only starting to learn Japanese. So without a translation service I would understand nothing. But it appears there are some services that let one understand it for the most part. Also I've found [a gist][1]. So with your explanation and the above mentioned web pages, I now know more, thanks :)

And I was wrong, `eskk` doesn't search the whole dictionary. It searches either the first part (if I tell it where the okurigana starts), or the second one. Also, I've found that instead of typing a capital letter, one can use `;` + a lowercase one.

[1]: https://gist.github.com/akirak/e93876855f8b1d295c2cd0ff2370af8d
